### PR TITLE
feat(model): 📈 rtmdet from mmyolo metrics are added

### PR DESCRIPTION
**Pull request by onuralpszr**
# Description

Rtmdet from mmyolo with metric results are added. In this installation I used python 3.9, pytorch 2.0 with cuda 118 and mmcv 2.0.0+cuda118

This is the order we have to install because mm* packages are asked to be "import torch" in installation torch needs to be install first. 

```console
pip install https://download.pytorch.org/whl/cu118/torch-2.0.0%2Bcu118-cp39-cp39-linux_x86_64.whl
pip install -U openmim
mim install "mmengine>=0.6.0"
pip install https://download.openmmlab.com/mmcv/dist/cu118/torch2.0.0/mmcv-2.0.0-cp39-cp39-manylinux1_x86_64.whl
mim install "mmdet>=3.0.0,<4.0.0"
```

-   [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by LinasKo** [comment]
I needed to install setuptools after step 2.

```bash
pip install https://download.pytorch.org/whl/cu118/torch-2.0.0%2Bcu118-cp39-cp39-linux_x86_64.whl
pip install -U openmim
pip install setuptools
mim install "mmengine>=0.6.0"
pip install https://download.openmmlab.com/mmcv/dist/cu118/torch2.0.0/mmcv-2.0.0-cp39-cp39-manylinux1_x86_64.whl
mim install "mmdet>=3.0.0,<4.0.0"
```

**Comment by LinasKo** [comment]
Author's results: https://github.com/open-mmlab/mmyolo/tree/main/configs/rtmdet

**Comment by onuralpszr** [comment]
> I needed to install setuptools after step 2.
> 
> ```shell
> pip install https://download.pytorch.org/whl/cu118/torch-2.0.0%2Bcu118-cp39-cp39-linux_x86_64.whl
> pip install -U openmim
> pip install setuptools
> mim install "mmengine>=0.6.0"
> pip install https://download.openmmlab.com/mmcv/dist/cu118/torch2.0.0/mmcv-2.0.0-cp39-cp39-manylinux1_x86_64.whl
> mim install "mmdet>=3.0.0,<4.0.0"
> ```

I added readme and just test it didn't asked setuptools but okay I am adding it.

**Comment by onuralpszr** [comment]
@LinasKo  I added setuptools as well

**Comment by onuralpszr** [comment]
```sh
pip install https://download.pytorch.org/whl/cu118/torch-2.0.0%2Bcu118-cp39-cp39-linux_x86_64.whl
pip install -U openmim
pip install setuptools
mim install "mmengine>=0.6.0"
pip install https://download.openmmlab.com/mmcv/dist/cu118/torch2.0.0/mmcv-2.0.0-cp39-cp39-manylinux1_x86_64.whl
mim install "mmdet>=3.0.0,<4.0.0"
mim install "mmyolo==0.6.0"
```

**Review by LinasKo — APPROVED** [review]
Confirming that code structure is correct, the results are within the range we expect, but I have no time to get my CUDA set up properly and won't be testing it.
